lansa-vmss-windows-autoscale-sql-database/azuredeploy.json

```diff
@@ -200,40 +287,43 @@
          "type": "string",
          "defaultValue": "0",
          "metadata": {
-            "description": "Re-run licensing"
+            "description": "Re-run licensing. It is unlikely that this parameter needs to be used"
          }
       }
    },
-  "variables": { 
-    "namingInfix": "[toLower(substring(concat(parameters('vmssName'), uniqueString(resourceGroup().id)), 0, 9))]",
-    "longNamingInfix": "[toLower(parameters('vmssName'))]",
-    "dblongNamingInfix": "[toLower(concat('db',parameters('vmssName')))]",
-    "dbvmssName": "[toLower(substring(concat('db',parameters('vmssName'), uniqueString(resourceGroup().id)), 0, 9))]",
+  "variables": {
+    "isNewDatabase": "[equals(parameters('databaseNewOrExisting'), 'new')]",
+    "namingInfix": "[toLower(substring(concat(parameters('stackName'), uniqueString(resourceGroup().id)), 0, 9))]",
+    "longNamingInfix": "[toLower(parameters('stackName'))]",
+    "dblongNamingInfix": "[toLower(concat('db',parameters('stackName')))]",
+    "aglongNamingInfix": "[toLower(concat('ag',parameters('stackName')))]",
+    "dbvmssName": "[toLower(substring(concat('db',parameters('stackName'), uniqueString(resourceGroup().id)), 0, 9))]",
```

Comment:
this won't be very likely to generate a unique name - i.e. if the stack name is > 6 chars it won't have any part of the uniqueString value

Reply:
See comment above on the MaxLength of 61